app/assets/javascripts/dropzone_input.js.coffee

```diff
@@ -133,24 +135,25 @@ class @DropzoneInput
     formatLink = (str) ->
       "![" + str.alt + "](" + str.url + ")"
 
-    handlePaste = (e) ->
-      e.preventDefault()
-      my_event = e.originalEvent
+    # Paste event handler
```

Comment:
Can you please remove this comment? The function name is already prety selfexplenatory :)